toolkit/tools/imagegen/configuration/systemconfig.go

```diff
@@ -76,6 +76,12 @@ func (s *SystemConfig) IsValid() (err error) {
 	//Validate PostInstallScripts
 	//Validate Groups
 	//Validate Users
+	for _, b := range s.Users {
```

Comment:
Suggestion: I'd first validate all users and then return a list of errors for each invalid one, so that the user has a clear picture of all misconfigured users in one run.

Reply:
Investigated this and determined this would not match how the errors are being presented in other areas. This would be best addressed in a future PR where all IsValid() errors for disk and systemconfig (and thus User) are shown at once. Also of note is that the ReMarshal tests won't support aggregated error messages since go's json.Unmarshal() bails when hitting first error.

Reply:
I see. In that case I agree this is out of scope for this PR. Thank you, @rlmenge.